h2o-r/tests/testdir_algos/xgboost/runit_INTERNAL_XGBoostEstimation.R

```diff
@@ -0,0 +1,71 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+
+
+## Data: 5MM records and 500 features
+
+CreateData <- function(nrows, ncols){
+  
+  airlines <- h2o.importFile("hdfs://mr-0xd6.0xdata.loc:8020/datasets/airlines_all.05p.csv")
```

Comment:
Should not be hardcoded. I think

Reply:
We have about 3 hadoop clusters with different name nodes.  Tests will fail if you hard code the path to your file.

Reply:
Ok. . .  I took this right from Megan's original code.  So, what should I change this to?